CMakeLists.txt

```
@@ -482,6 +482,19 @@ if (OPENMC_USE_MPI)
   target_link_libraries(libopenmc MPI::MPI_CXX)
 endif()
 
+#===============================================================================
+# Log build info that this executable can report later
+#===============================================================================
+target_compile_definitions(libopenmc PRIVATE BUILD_TYPE=${CMAKE_BUILD_TYPE})
+target_compile_definitions(libopenmc PRIVATE COMPILER_ID=${CMAKE_CXX_COMPILER_ID})
+target_compile_definitions(libopenmc PRIVATE COMPILER_VERSION=${CMAKE_CXX_COMPILER_VERSION})
+if (OPENMC_ENABLE_PROFILE)
+    target_compile_definitions(libopenmc PRIVATE PROFILINGBUILD)
```

Just to be consistent with spacing elsewhere in this file
```suggestion
  target_compile_definitions(libopenmc PRIVATE PROFILINGBUILD)
```